spec/factories/blocks.rb

```diff
@@ -0,0 +1,19 @@
+FactoryGirl.define do
+  factory :block do
+    title 'Block 1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.